libs/sdk-ui-filters/api/sdk-ui-filters.api.md

```diff
@@ -547,6 +548,8 @@ export interface IAttributeFilterDropdownBodyProps {
 
 // @beta
 export interface IAttributeFilterDropdownButtonProps {
+    attributeDataSet?: IMetadataObject;
```

Comment:
I am not sure about providing attributeDataSet and shouldDisplayAttributeTooltip as props. I thinking about scenario that user will customise content of tooltip completely. If we move loading dataset to parent. maybe its good just provide tooltip content component.  Here AttributeFilterDetail. 

Pls I know that @ivan-nejezchleb is preparing single select attribute pls sync with him.